php/Log/NullLogger.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PHPCD\Log;
+
+use Psr\Log\AbstractLogger;
+
+class NullLogger extends AbstractLogger
+{
+    /**
+     * Does nothing
+     *
+     * @param mixed  $level
+     * @param string $message
+     * @param array  $context
+     *
+     * @return null
+     */
+    public function log($level, $message, array $context = [])
+    {
+        return null;
```

Comment:
1. In php, a function without return statement will return null actually
2. The name `NullLogger` has indicated that this logger will do nothing.
3. `return null` can not prevent users to add some other code to the log method.

All in all, an empty log function is enough.